src/components/Operatives/NewTaskForm.js

```diff
@@ -72,7 +80,9 @@ const NewTaskForm = ({ workOrderReference }) => {
         requestData: workOrderUpdateFormData,
       })
 
-      router.push(`/work-orders/${workOrderReference}`)
+      router.push(
+        `/operatives/${currentUser?.operativePayrollNumber}/work-orders/${workOrderReference}`
```

Comment:
Unsure that using a safe navigation operator here is wise - wouldn't it be better to raise an error?